### DISCUSSION TITLE
Version 3.1.3

### DISCUSSION THREAD

**Discussion by SteffenBernd** [netbox-community/netbox-docker#683] [General]
Hey,
when will the netbox version 3.1.3 be released?
And how long do you stay behind in versions with this docker? Is there a ETA like netbox releases a new version, we wait 1-2 months and then upgrade the docker compose?
Greetings
Steffen

**Comment by tobiasge** [comment]
The tags for Netbox 3.1.3 were pushed 4 days ago. Builds are done every night. So the tags are usually published the days after an new Netbox release.

**Comment by SteffenBernd** [comment]
i just docker-compose pull ed. Why do i have this version (v3.0.11)?
should this not be 3.1.3?

**Comment by tobiasge** [comment]
Check if have pinned the version in your `docker-compose.override.yml` or if you have the correct Git tag.

**Comment by SteffenBernd** [comment]
This is my override

version: '3.4'
services:
  netbox:
    restart: unless-stopped
    ports:
      - 8001:8080


do i need to change that version: '3.4' to something else?

**Comment by tobiasge** [comment]
Then the version is not pinned in the override file. 
I think you have an old Git tag und should do an update of the repository.

**Comment by SteffenBernd** [comment]
and how do i do that? im kinda new to this and a linux noob ;)

**Comment by tobiasge** [comment]
We have that documented in our [Wiki](https://github.com/netbox-community/netbox-docker/wiki/Updating).

Also make sure to read all release notes for Netbox Docker and Netbox itself.

**Comment by SteffenBernd** [comment]
when i follow the update guide i get this and netbox does not start again
![image](https://user-images.githubusercontent.com/92783384/147941870-003d5af4-d5c8-420c-94e8-a5a77c661144.png)

**Comment by tobiasge** [comment]
I think you didn't upgrade the Postgres properly with those [instructions](https://github.com/netbox-community/netbox-docker/wiki/Updating#postgresql-update).
That's why you need to read all the release notes.

**Comment by SteffenBernd** [comment]
this link does not work.
if you are refering to the postgress update, dosnt the pull upgrade that as well?

**Comment by SteffenBernd** [comment]
Ok not it works. i really had to manually update postgres.
thx for your help :)
